### DISCUSSION TITLE
Add prefix to Console

### DISCUSSION THREAD

**Discussion by arabnejad** [Textualize/rich#1312] [Q&A]
Hi 

is there any way to add a prefix (like what we have within `log`) for `Console` class, so all print output will the passed prefix string at first

unfortunately, I can not use log for those cases :(

**Comment by willmcgugan** [comment]
I don't follow. Can you give me an example of what you are looking for?

**Reply by arabnejad** [reply]
Hi @willmcgugan 
Thanks for your quick reply :)

for example
```python
# adding prefix to Console and set it as the active one
print("test string1")
print("test string2")
# disable prefix or set prefix=""
print("test string3")
```
`output` :
```sh
[test@localhost] test string1
[test@localhost] test string2
test string3
```

I already added a wrapper to rich print function, 
I was wondering if we can have this as the pre-defined functionality or not

**Reply by willmcgugan** [reply]
There’s nothing built in to do that. Sounds like your wrapper is a good enough solution…
